src/backend/aspen/api/schemas/samples.py

```diff
@@ -1,12 +1,16 @@
 from pydantic import constr
 
-from aspen.api.schemas.base import BaseRequest
+from aspen.api.schemas.base import BaseRequest, BaseResponse
 
 
 class SampleRequestSchema(BaseRequest):
     # mypy + pydantic is a work in progress: https://github.com/samuelcolvin/pydantic/issues/156
     name: constr(min_length=1, max_length=128, strict=True)  # type: ignore
 
 
-class SampleResponseSchema(BaseRequest):
+class SampleResponseSchema(BaseResponse):
     name: str
+
+
+class SampleDeleteResponse(BaseResponse):
+    id: int
```

Comment:
You know, after thinking about it some more, as much as I like the idea of having `id` very consistently mean the PK, I don't know if it's actually doable what with all the model inheritance stuff we have going on.

Reply:
It is. Most/all of our endpoints are currently scoped to specific types of entities/workflows (in total misalignment with our db schema) so I think this will work fine.